web/src/click_handlers.ts

```diff
@@ -983,6 +983,33 @@ export function initialize(): void {
         }
     });
 
+    const compose_buttons = tippy.default(".compose_button_tooltip", {
+        trigger: "mouseenter",
+        placement: "top",
+    });
+
+    tippy.createSingleton(compose_buttons, {
+        onTrigger(instance, event) {
```

Comment:
Let's also define the `delay` property for the singletons over here and set it to `delay: LONG_HOVER_DELAY`.